example/20-componentconfig-gardenlet.yaml

```diff
@@ -125,6 +125,7 @@ featureGates:
   ShootSARotation: true
   HAControlPlanes: true
   DefaultSeccompProfile: true
+  CoreDNSQueryRewriting: true
```

Comment:
You could also add the feature gate to the following example file `example/gardener-local/gardenlet/values.yaml`.

Reply:
Thanks for point out.
Addressed with latest commit.